src/cli/commands/identity/status.ts

```diff
@@ -0,0 +1,40 @@
+import { DateTime } from "luxon";
+import { CommandModule } from "yargs";
+import { BaseCommand, ConfigFileOptions, configOptionBuilder } from "../../BaseCommand";
+
+export const identityStatusHandler = async ({ config }: ConfigFileOptions): Promise<void> => {
+    await new IdentityStatus().run(config);
+};
+export const yargsIdentityStatusCommand: CommandModule<{}, ConfigFileOptions> = {
+    command: "status",
+    describe: "show the status of the identity",
+    handler: identityStatusHandler,
+    builder: configOptionBuilder
+};
+
+export class IdentityStatus extends BaseCommand {
+    protected async runInternal(): Promise<void> {
+        await this.createRuntime();
+        if (!this.cliRuntime) {
+            throw new Error("Failed to inizitialize runtime");
+        }
+
+        try {
+            const identityInfoResult = await this.cliRuntime.getServices().transportServices.account.getIdentityInfo();
+            const identityDeletionProcessesResult = await this.cliRuntime.getServices().transportServices.identityDeletionProcesses.getActiveIdentityDeletionProcess();
```

Comment:
Please change variable name to use singular instead of plural (there can be at most one active IdentityDeletionProcess as far as I know).